x-pack/plugins/siem/public/endpoint_policy/view/policy_list.tsx

```diff
@@ -44,18 +51,48 @@ const PolicyLink: React.FC<{ name: string; route: string; href: string }> = ({
   );
 };
 
+// eslint-disable-next-line react/display-name
```

Comment:
@paul-tavares essentially took this from some ingest code, thoughts on just importing their component directly?

Reply:
@kevinlog  I probably would not do that (import directly) for a few reasons. We don't want to couple our display needs to the Ingest component since that would cause any change on the Ingest side to also have a direct effect on our side. The other reason would be the possibility of inflating our app's bundle during compile, since it would likely bring it would bring in their component along with any dependency it has.

Reply:
one other thing: This component (`TableRowAction`) was needed on the Ingest side (if I remember correctly) because of how their code works in handling actions (they use a React context "provider" wrapper component). For us, this might not be needed. `EuiBasicTable` already supports collapsing actions into a menu auto-magically. Check out their examples/docs